### PR TITLE
21161-Epicea-do-not-create-directory-on-startUp-if-EpMonitor-is-not-logging

### PR DESCRIPTION
**Pull request by VincentBlondeau**
Resolve https://pharo.fogbugz.com/f/cases/21161/Epicea-do-not-create-directory-on-startUp-if-EpMonitor-is-not-logging#BugEvent.196160

### CODE REVIEW

File: src/OmbuTests/OmSessionStoreNameStrategyTest.class.st [hunk 1]
```diff
@@ -41,7 +41,7 @@ OmSessionStoreNameStrategyTest >> fileReferenceWith: aName [
 
 { #category : #running }
 OmSessionStoreNameStrategyTest >> setUp [
-	super setUp.
```
Comment (jecisc) [inline src/OmbuTests/OmSessionStoreNameStrategyTest.class.st]: I think this change is unintended.

**Review by jecisc — CHANGES_REQUESTED** [review]

File: src/OmbuTests/OmSessionStoreNameStrategyTest.class.st (hunk 1 above)
Comment (VincentBlondeau) [reply src/OmbuTests/OmSessionStoreNameStrategyTest.class.st]: indeed! Updated ;)

**Review by jecisc — APPROVED** [review]

File: src/OmbuTests/OmSessionStoreNameStrategyTest.class.st [hunk 2]
```diff
@@ -54,8 +54,8 @@ OmSessionStoreNameStrategyTest >> strategyClass [
 
 { #category : #running }
 OmSessionStoreNameStrategyTest >> tearDown [
-
-	self directory ensureDeleteAll 
+	self directory ensureDeleteAll.
+	super tearDown
```
[the remark this answers is not in the document]
Comment (jecisc) [reply src/OmbuTests/OmSessionStoreNameStrategyTest.class.st]: For a fix this small I think it should be ok :)

[the remark this answers is not in the document]
Comment (astares) [reply src/OmbuTests/OmSessionStoreNameStrategyTest.class.st]: Be aware of https://github.com/pharo-project/pharo/pull/778 and https://pharo.fogbugz.com/f/cases/21103/Super-tearDown-need-to-be-called-as-last-message-in-tearDown-of-Ombu-tests

**Comment by VincentBlondeau** [comment]
I updated the test tearDown

**Review by tinchodias — APPROVED** [review]

**Comment by MarcusDenker** [comment]
now we have a conflict:

Conflicting files
src/OmbuTests/OmSessionStoreNameStrategyTest.class.st

**Comment by astares** [comment]
Yes - because as you can see on the diff OmSessionStoreNameStrategyTest >> tearDown  is still included (because still changed)

**Comment by MarcusDenker** [comment]
What is strange is that another issue now is constantly failing due to a test failing in
OmSessionStoreTest

**Comment by MarcusDenker** [comment]
can it be that the tearDown change is problematic? Since we merged that, we have always OmDeferrerTest and OmSessionStoreTest failing

**Comment by astares** [comment]
Yes it was already merged and is also not related to this bug. It should be reverted to the original one so you have no diff at all on that method. Otherwise you will have a conflict.

**Comment by tinchodias** [comment]
I wonder what's the reason to force #tearDown as last message

**Comment by jecisc** [comment]
@tinchodias Check the discussion here: 

https://github.com/pharo-project/pharo/pull/767

**Pull request merged**